src/safeds_stubgen/api_analyzer/_api.py

```diff
@@ -252,9 +261,11 @@ class Parameter:
     name: str
     is_optional: bool
     default_value: str | bool | int | float | None
+    default_is_none: bool
```

Comment:
Why do we need this? Isn't the combination of `is_optional` and `default_value` enough?

Reply:
Removed this field.